server/desktop/src/main/java/dev/slimevr/desktop/tracking/trackers/hid/TrackersHID.kt

```diff
@@ -236,13 +235,15 @@ class TrackersHID(name: String, private val trackersConsumer: Consumer<Tracker>)
 						// x y z w -> w x y z
 						var rot = Quaternion(q[3].toFloat(), q[0].toFloat(), q[1].toFloat(), q[2].toFloat())
 						val scaleRot = 1 / (1 shl 15).toFloat() // compile time evaluation
-						rot = AXES_OFFSET.times(scaleRot).times(rot) // no division
+						rot *= scaleRot // no division
+						rot = axisOffset(rot)
```

Comment:
Why not just do it in one line
```suggestion
						rot = axisOffset(rot * scaleRot) // no division
```